solutions/add_two_numbers_test.ts

```diff
@@ -0,0 +1,10 @@
+import { test } from "https://deno.land/std/testing/mod.ts";
+import { assertEquals } from "https://deno.land/std/testing/asserts.ts";
+import { createSinglyLinkedListNode } from '../test_utilities/linked_list.ts';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/axross/leetcode-typescript/pullRequest?prid=7152801)
```suggestion
import { createSinglyLinkedListNode } from "../test_utilities/linked_list.ts";
```